ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -48,6 +48,33 @@
     }
   }
 
+  &__more-options-beta {
+    margin-top: calc($spacer-unit / 2);
+
+    a {
+      color: $color-dark-blue;
```

Comment:
I think we will need to add the :visited styling on this so that we retain the blue - I don't think we should style this kind of link differently if you've viewed this page.

Reply:
@jlhdxw thank you this had visited but was in color black so updated to color dark blue